blob/fileblob/fileblob.go

```diff
@@ -176,6 +177,11 @@ type Options struct {
 	// If true, create the directory backing the Bucket if it does not exist
 	// (using os.MkdirAll).
 	CreateDir bool
+
+	// SkipMetadata will skip the writing of any metadata files (sidecar files).
+	// Metadata read will be incomplete at best and should not be relied upon
+	// more than that of a ordinary file system's.
+	SkipMetadata bool
```

Comment:
This is only skipping metadata write, so perhaps it would be more clear to name it `SkipMetadataWrites`? Or maybe it's clear enough from the docstring.

Reply:
We can always go with a stronger guarantee (in place of “metadata read will be inconsistent and should not be relied upon”) later.

The idea is to retain the option to override or set metadata if need be, at the expense of futile reads—if this were not wanted, `http.FileServe` would fill that niche.